src/main.js

```diff
@@ -5,17 +5,32 @@ import { Accordion, initializeAccordion } from './components/Accordion/Accordion
 
 import { AddToDoField, initializeAddToDoField } from './components/AddToDoItem/AddToDoField';
 
+import { getTasksDataFromLocalStorage } from './helpers/storage';
+
+import { ToDoItem } from './components/ToDoItem/ToDoItem';
+
+
+
 // selectors
 const pageTitle = document.querySelector('#title');
 const accordion = document.querySelector('#accordion');
-const addToDoSection1 = document.querySelector('#todo-section');
+const addToDoSection = document.querySelector('#todo-section');
+
+// load data from local storage and render tasks
+const dataOnLoad = getTasksDataFromLocalStorage();
+export const renderTasks = () => {
+  dataOnLoad.forEach((task) => {
+    addToDoSection.prepend(ToDoItem(task.name, task.id, task.complete));
```

Comment:
you can do it this way
`addToDoSection.prepend(ToDoItem(...task))` OR
better to have ToDoItem which will take as an argument object with properties {name:'', id'', checked}
then you would simply pass this task as 
` addToDoSection.prepend(ToDoItem(task)`

Reply:
.